images/README.md

```diff
@@ -1,25 +1,9 @@
-# build-env
-
-Docker environment used to build official images.
-
-This used be here but now lives at the repo attached to
-[tyk-build-env](https://hub.docker.com/r/tykio/tyk-build-env)
-
 # plugin-compiler
 
-The usecase is that you have a plugin (probably Go) that you require
-to be built.
-
-Navigate to where your plugin is and build using a docker volume to
-mount your code into the image. Since the vendor directory needs to be
-identical between the gateway build and the plugin build, this means
-that you should pull the version of this image corresponding to the
-gateway version you are using.
-
-This also implies that if your plugin has vendored modules that are
-[also used by Tyk
+Does not support go modules. If your plugin has vendored modules that
+are [also used by Tyk
 gateway](https://github.com/TykTechnologies/tyk/tree/master/vendor)
-then your module will be overridden by the version that Tyk uses. 
+then your module will be overridden by the version that Tyk uses.
```

Comment:
This repo doesn't have a `vendor` dir anymore and uses Go modules exclusively :see_no_evil: